example-apps/tests-cucumber/exoservice/README.md

```diff
@@ -0,0 +1,2 @@
+# 
+> 
```

Comment:
Lets make the example app have the minimum needed for testing. I believe we can remove this file